master/Makefile

```diff
@@ -221,3 +230,70 @@ publish-nvcr:
 		docker tag $(DOCKER_REPO)/$(PROJECT_NAME):$(FULL_COMMIT)-amd64 $$image; \
 		docker push $$image; \
 	done
+
+# Proto stuff
+# String constants.
```

Comment:
I would suggest putting the proto stuff first, since it is a more fundamental dependency